test/chplenv/chplconfig/correctness/correctness.good

```diff
@@ -0,0 +1,17 @@
+CHPL_TARGET_PLATFORM: darwin *
```

Comment:
I'm not following how these tests work, will need to bug you in person about them